.github/CODEOWNERS

```diff
@@ -1,7 +1,7 @@
 # Require review from golang-compiler team for changes in any file. This keeps us in the loop on
 # auto-merge PRs. The review bot is also an owner so that it can still trigger auto-merge for sync
 # PRs on its own. We may remove this rule once auto-merges are routine.
-* @microsoft/golang-compiler @microsoft-golang-review-bot
+* @microsoft/golang-compiler @review-bot-for-go
```

Comment:
Mentioned in IM, but here for context: this doesn't seem like it'll work: https://github.com/orgs/community/discussions/23064. There is a workaround: explicitly list all the files that the app will touch and assign no code owner.

Reply:
Ah, actually the file viewer directly says it won't work:

![image](https://github.com/user-attachments/assets/c1d50248-a8ee-454c-8ebc-7b85adad8a3d)